src/main/webapp/app/exercises/programming/participate/code-editor-student-container.component.html

```diff
@@ -35,7 +35,7 @@
                 ></jhi-included-in-score-badge
             ></span>
         </div>
-        <div submissionPolicy *ngIf="exercise.submissionPolicy && numberOfSubmissionsForSubmissionPolicy !== undefined">
+        <div submissionPolicy *ngIf="exercise.submissionPolicy && exercise.submissionPolicy.active && numberOfSubmissionsForSubmissionPolicy !== undefined">
```

Comment:
```suggestion
        <div submissionPolicy *ngIf="exercise.submissionPolicy?.active && numberOfSubmissionsForSubmissionPolicy !== undefined">
```

Reply:
I tried that but the linter is sadly not as smart in the html template and complains later on that the submission policy might be undefined